_data/init-cli/docker_init.yaml

```diff
@@ -0,0 +1,151 @@
+command: docker init
+short: Creates Docker-related starter files for your project
+long: |-
+  Initialize a project with the files necessary to run the project in a container.
+
+  Docker Desktop 4.18 and later provides the Docker Init plugin with the `docker init` CLI command. Run `docker init` in your project directory to be walked through the creation of the following files with sensible defaults for your project:
+    
+    * .dockerignore
+    * Dockerfile
+    * docker-compose.yaml
+    
+    If any of the files already exist, a prompt appears and provides a warning as well as giving you the option to overwrite all the files.
+    
+    > **Warning**
+    >
+    > You can't recover overwritten files.
+    > To back up an existing file before selecting to overwrite it, rename the file or copy it to another directory.
+    {: .warning}
+  
+  After running `docker init`, you can choose one of the following templates:
+    
+    * Go: Suitable for a Go server application.
+    * Other: General purpose starting point for containerizing your application.
+  
+  After `docker init` has completed, you must modify the created files and tailor them to your project. Visit the following topics to learn more about the files:
+    
+    * [.dockerignore](../../../engine/reference/builder.md#dockerignore-file)
+    * [Dockerfile](../../../engine/reference/builder.md)
+    * [docker-compose.yaml](../../../compose/compose-file/03-compose-file.md)
+
+usage: docker init [OPTIONS]
+pname: docker
+plink: docker.yaml
+options:
+    - option: version
+      shorthand: v
```

Comment:
@rafd123 do you know if the shorthand `-v` has been discussed? We try to avoid shorthands if there's chances they become ambiguous. In this case `-v` could collide with `--verbose` on some other commands (which could potentially be an option we might add to `docker init` in future, and which would be more commonly used by users than `--version`).

Reply:
^^ If we're unable to update the binary, we could decide to just remove it here from the docs for now)

Reply:
It hasn't come up. FWIW, `docker init` was modeled after `docker dev`, which includes a shorthand for `--version`: https://github.com/docker/docker-dev/blob/main/docs/reference/dev.md

Whether we update the binary to exclude it is a function of whether it's worth cutting a new 4.18 release, I *think*.

Reply:
With the understanding that it has no bearing on the 4.18 release, any objection to me just removing the `-v` shorthand from `docker init` now so we don't forget?

Reply:
Seems fine to me!

(I can also give some pointers on how to make it generate the page as it is now for future builds)